tests/test_data/tableanddish.com/tableanddish.json

```
@@ -0,0 +1,33 @@
+{
+  "author": "Sally Roeckell for Table and Dish",
+  "canonical_url": "https://www.tableanddish.com/swordfish-piccata/",
+  "site_name": "TableandDish",
+  "host": "tableanddish.com",
+  "language": "en-US",
+  "title": "Swordfish Piccata",
+  "ingredients": [
+    "1/2 cup all-purpose flour for dredging",
+    "1 teaspoon sea salt",
+    "1 teaspoon fresh black pepper",
+    "3 fresh swordfish steaks",
+    "1 tablespoon extra virgin olive oil",
+    "2 tablespoon unsalted butter",
+    "4 cloves garlic, thinly sliced",
+    "1/3 cup fresh lemon juice",
+    "1/2 cup dry white wine",
+    "2 tablespoons capers, drained",
+    "1/3 cup minced fresh parsley",
+    "Lemon garnish"
+  ],
+  "instructions_list": [
+    "Combine the flour, pepper, and salt in a shallow dish such as a pie plate. Dredge the swordfish steaks in the flour mixture and shake off any excess.",
+    "In a large skillet, over medium-high heat, heat the olive oil with the butter. When hot, add the fish and cook until browned on the underside, 2 to 3 minutes. Turn fish over and cook until well browned on the other side. Transfer to a platter and keep warm.",
+    "To make the sauce:",
+    "Add the garlic to the skillet and cook until fragrant, 30 seconds to 1 minute.",
+    "Add the lemon juice and wine to the pan and deglaze, scraping up any browned bits. Bring to a boil, turn heat to low and add the capers. Adjust the salt seasoning to taste.",
+    "Return the swordfish to the skillet and let the fish cook for a few minutes so that it can absorb the flavors of the sauce. Sprinkle with the parsley and serve at once, garnished with parsley and lemon."
+  ],
+  "category": null,
```

```suggestion
  "yields": null,
```

Since `category` is an [optional](https://github.com/hhursev/recipe-scrapers/blob/main/docs/contributing/in-depth-guide-scraper-functions.md#optional-functions) field we can exclude it from the test output. However `yields` and `total_time` are [mandatory](https://github.com/hhursev/recipe-scrapers/blob/main/docs/contributing/in-depth-guide-scraper-functions.md#mandatory-functions) so we can include them as `null` since the information is not available on the site.